ledger-core/virtual/execute/execute.go

```diff
@@ -1001,6 +1002,54 @@ func (s *SMExecute) stepFinishRequest(ctx smachine.ExecutionContext) smachine.St
 	return ctx.Stop()
 }
 
+func (s *SMExecute) stepSendExistingCallResult(ctx smachine.ExecutionContext) smachine.StateUpdate {
```

Comment:
why don't you use stepSendCallResult like we do in other places to send result?

Reply:
stepSendCallResult goes to stepFinishRequest and further. As I understand we should only send results and stop.

Reply:
I see, still think we can re-use some code from there...

Reply:
We can extract send code in a function, if you wish?